rollup.config.js

```diff
@@ -74,6 +80,28 @@ const configs = [
     plugins: commonPlugins,
   },
 
+  // React Native
+  {
+    input: inputFile,
+    output: {
+      file: `native/recoil.js`,
+      format: 'es',
+      exports: 'named',
+    },
+    external: [...externalLibs, 'react-native'],
+    plugins: commonPlugins.map(plugin => {
+      // Replace the default nodeResolve plugin
```

Comment:
I'm not a huge fan of the way this looks, but it works. Another option would be to turn commonPlugins into a function that takes in `nodeResolve` options.

Reply:
what happens if we have `extensions: ['.native.js', '.js']` for other builds?

Reply:
Adding this will replace any file named `x.js` with its native counterpart (`x.native.js`) if that native counterpart exists. So we should only specify this extensions rule for when we want the bundle to include code meant for RN.